network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -1166,10 +1168,11 @@ class GeneralProductPage(ProductPage):
 
     # How can you control your data
 
-    how_can_you_control_your_data = models.TextField(
+    how_can_you_control_your_data = RichTextField(
         max_length=5000,
+        features=['bold', 'italic', 'link'],
```

Comment:
Upon request of Pomax, we have created a ticket to unify these choices and have them in a new list, which we can call whenever making/updating a rich text field.

https://github.com/mozilla/foundation.mozilla.org/issues/7755